ecs-agent/netlib/platform/common_linux.go

```diff
@@ -695,22 +695,31 @@ func (c *common) configureBranchENI(ctx context.Context, netNSPath string, eni *
 		"NetNSPath": netNSPath,
 	})
 
-	var cniNetConf ecscni.PluginConfig
+	c.os.Setenv(IPAMDataPathEnv, filepath.Join(c.stateDBDir, IPAMDataFileName))
```

Comment:
why did we add this line? Can you please add a comment as well?

Reply:
Will do. This set the path for the IPAM log files, the default is a path in  the root volume, this will fail in OS Bottlerocket because the root vol is read-only.